event/src/Event/Domain/Command/FinishEvent.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Prooph\Common\Messaging\Command;
+
+use Event\Domain\Value\EventId;
+use Event\Domain\Value\EventType;
+
+
+final class FinishEvent extends Command
+{
+
+    /**
+     * @var \Event\Domain\Value\EventId
+     */
+    private $id;
+
+    /**
+     * @var \Event\Domain\Value\EventType
+     */
+    private $type;
+
+
+    private function __construct(EventId $id, EventType $type)
+    {
+        $this->init();
+
+        $this->id = $id;
+        $this->type = $type;
+    }
+
+    public static function fromFeedEntry(EventId $id, EventType $type) : self
+    {
+        return new self($id, $type);
+    }
+
+    /**
+     * @return EventId
+     */
+    public function id(): EventId
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return EventType
+     */
+    public function type(): EventType
+    {
+        return $this->type;
+    }
+
+
+
+    protected function setPayload(array $payload): void
+    {
+        /**
+         * @var EventType
```

Comment:
No longer needed as per latest `master`, but not super-important to fix right now

Reply:
setPayload removed